COPYING.LESSER

```diff
@@ -0,0 +1,165 @@
+                   GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
According to the official guidelines, both the GPL and the LGPL license text should be included: https://www.gnu.org/licenses/gpl-howto.en.html The files should be called `COPYING` and `COPYING.LESSER`.